catalog/spatial/registry/registry-federation-admin-service-impl/src/main/java/org/codice/ddf/registry/federationadmin/service/impl/IdentityNodeInitialization.java

```diff
@@ -149,6 +162,7 @@ private void createIdentityNode() throws FederationAdminException {
 
             federationAdminService.addRegistryEntry(identityMetacard);
         }
+        LOGGER.info("Successfully create registry identity node: {}", registryPackageId);
```

Comment:
Changed to created.
I think info works here as it's giving a general heads up that registry that was just started now has an identity metacard. Displaying the registryId gives it a debug feel.  This should only log once as there should only be one identity node per system. 
I can change it to debug if you think it should be.